_data/other_converters.yml

```diff
@@ -0,0 +1,31 @@
+# template
+#     - name:
+#       url: URL to repo
+#       data_types: MRI, EEG, MEG, iEEG, events
+#       expected_input: DICOM, PAR/REC, tsv
+#       language: python, matlab, docker
+#       documentation: URL to documentation
+#       comment:
+#       GUI: false
+
+- group:
```

Comment:
The content of this file is not being rendered on the website but I am keen to keep it around so that we have it filed and accessible somewhere. Does that make sense?